cranecloud/docs/authentication/forgotPassword.md

```diff
@@ -0,0 +1,16 @@
+## Forgot Password
+**User Actions:**
+
+Once you click the forgot password link on the login page, it will lead you to this route http://cranecloud.io/forgot-password that displays a page for you to enter your email address.
```

Comment:
link changed to <http://staging.cranecloud.io/login>

Reply:
This too is fixed now